app/controllers/miq_ae_customization_controller/old_dialogs.rb

```diff
@@ -250,7 +250,7 @@ def old_dialogs_update_create
       if @edit[:new][:name].blank?
         add_flash(_("Name is required"), :error)
       end
-      unless @edit[:new][:dialog_type]
+      if @edit[:new][:dialog_type].blank?
         add_flash(_("Dialog Type must be selected"), :error)
       end
       begin
```

Comment:
should we wrap https://github.com/bmclaughlin/manageiq/blob/c67e5c1e88e76ff82c9cb3531d3043690f77de32/app/controllers/miq_ae_customization_controller/old_dialogs.rb#L256-L260 with ``` unless @flash_array ``` to skip ``` YAML.parse(@edit[:new][:content]) ``` if there are errors we should just render flash message.